test/gurka/test_search_filter.cc

```diff
@@ -1057,7 +1057,7 @@ TEST_P(ClosuresWithTimedepRoutes, IgnoreClosureWithTimedepReverse) {
                                     {"/date_time/value", "current"},
                                     {costing_speed_type, "current"}},
                                    reader);
-    gurka::assert::raw::expect_path(result, {"AB", "BC", "CD", "DE", "EF", "FG", "GH"});
+    gurka::assert::raw::expect_path(result, {"BC", "CD", "DE", "EF", "FG", "GH"});
```

Comment:
@danpat I remember you mentioning that one of the timedep variants was counting an extra edge, was that for timedep reverse? Looking at the original test, I dont know why it included `AB` even though the route was from `B -> H`. It actually looks correct now!

Reply:
Yeah, this was the same thing - timedep_reverse wasn't stopping until it found the node prior to the actual one you wanted.  I guess reverse search was little-enough used that this oddness in results didn't tickle anyone's curiosity enough to dig into it fully.

Until now!